src/ui/Facet/Facet.ts

```diff
@@ -149,7 +149,7 @@ export class Facet extends Component {
      * Specifies the index field whose values the facet should use.
      *
      * This requires the given field to be configured correctly in the index as a *Facet field* (see
-     * [Adding Fields to a Source](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=137)).
+     * [Adding Fields to a Source](https://docs.coveo.com/en/1947/)).
```

Comment:
This is a Coveo Cloud V1 link; use [Add or Edit Fields](https://docs.coveo.com/1982/) instead